.travis.yml

```diff
@@ -9,6 +9,7 @@ env:
   - DJANGO_VERSION: "'Django~=1.11.0'"
   - DJANGO_VERSION: "'Django~=2.1.0'"
   - DJANGO_VERSION: "'Django~=2.2.0'"
+  - DJANGO_VERSION: "'Django~=3.0'"
```

Comment:
Let's write it as `3.0.0` 
It might not be the same thing.

Also, please see https://github.com/sobolevn/django-split-settings/issues/120#issuecomment-561059215

Reply:
Sure. I write 3.0 because this is how it shows PyPi. I'll fix all the issues.